modules/home/my_home/default.nix

```diff
@@ -24,114 +23,111 @@ with lib;
       };
     };
   };
-    config = mkIf (cfg.username != "") {
-      home-manager = {
```

Comment:
Home Manager modules are given to Home Manager directly. This means that you're dealing with the Home Manager module system directly and not the NixOS/nix-darwin ones. So options will be these directly: https://nix-community.github.io/home-manager/options.html